src/osd/scrubber/pg_scrubber.h

```diff
@@ -794,7 +796,7 @@ class PgScrubber : public ScrubPgIF,
    *
    * The selected range is set directly into 'm_start' and 'm_end'
```

Comment:
Function comment should indicate *exactly* how the caller should interpret the return value.